src/components/resumes/ResumesAPI.js

```diff
@@ -19,3 +19,78 @@ export async function postResume(resume) {
     return { ok: false };
   }
 }
+
+export default async function get() {
+  let errorRevise = false;
+  try {
+    let { data: result } = await axios.get(
+      "https://jsonplaceholder.typicode.com/users"
+    );
+    result = [
+      {
+        // Testing that the list is displayed with the requested information
+        address: {
+          city: "Bolivia",
+          street: "America",
+        },
+        contact: {
+          email: "filibertoAlarcon@gmail.com",
+        },
+        creationDate: "2021-09-03T14:56:28.000Z",
+        email: "segundo corre",
+        id: 1,
+        lastUpdateDate: "2006-05-22T17:14:03.000Z",
+        phone: "1-770-736-8031 x56442",
+        title: "Filiberto Alarcon",
+        username: "Parkt",
+        website: "geo-industri.org",
+      },
+
+      {
+        address: {
+          city: "Bolivia",
+          street: "Aroma",
+        },
+        contact: {
+          email: "Juan.Perez@gmail.com",
+        },
+        creationDate: "2008-09-13T09:21:59.4586",
+        email: "segundo corre",
+        id: 1,
+        lastUpdateDate: "2015-05-22T09:21:01.000Z",
```

Comment:
why are you calling the jsonplaceholder api first and then changing the result to fake data array?
do you really need to call that api?